### PR TITLE
add colorize applied plugin

### PR DESCRIPTION
**Pull request by cappyzawa**
Hi.
I implemented a plugin to help confirm the applied results.

ss 2022-08-19 23 55 14

- [x] Make sure you read the Plugin Naming Guide: https://krew.sigs.k8s.io/docs/developer-guide/develop/naming-guide/
- [x] Verify you can install your plugin locally: kubectl krew install --manifest=[...] --archive=[...]
```
kubectl krew install --manifest=plugins/colorize-applied.yaml
Installing plugin: colorize-applied
Installed plugin: colorize-applied
\
 | Use this plugin:
 |      kubectl colorize-applied
 | Documentation:
 |      https://github.com/cappyzawa/kubectl-colorize-applied
/
```
-->

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/colorize-applied.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: colorize-applied
+spec:
+  version: v0.0.1
+  homepage: https://github.com/cappyzawa/kubectl-colorize-applied
+  shortDescription: "Colorize the applied result"
```
Comment (ahmetb) [inline plugins/colorize-applied.yaml:8]: Colorize results of apply/dry-run

File: plugins/colorize-applied.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  description: |
+    Enjoy coloring the applied results.
```
Comment (ahmetb) [inline plugins/colorize-applied.yaml:10]: This doesn't help with anything. 

Please explain what the plugin does as if the screenshot doesn't exist.

**Review by ahmetb — CHANGES_REQUESTED** [review]
Thanks for the submission.

File: plugins/colorize-applied.yaml (hunk 2 above)
Comment (cappyzawa) [reply plugins/colorize-applied.yaml]: Thank you for your review!
I fixed it. (https://github.com/kubernetes-sigs/krew-index/pull/2522/commits/28f35b88ef0ffbd8e5421b160b8ff88a9df5d149) .

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Pull request merged**